docs/scos/dev/upgradability-services/new-documentation/private-api-class-extended-or-used.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Private API class was extended or used
+description: Reference information for evaluator and upgrader tools.
+last_updated: Mar 23, 2022
+template: concept-topic-template
+---
+
+## Private API class was extended or used
+
+Private API updates can break backward compatibility. Link for API's documentation - [see here]{link to "Private API class was extended or used" - private-api-class-extended-or-used.md#Private API class was extended or used}
+
+The following core classes are exceptions, and you can use and extend them on the project level:
```

Comment:
Remove the whole block from here. 
These block has to be part of privatePublic API docuemntation

Reply:
Blocks to remove:
"All the classes from the modules:"
"Particular classes:"